invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/message.overrides

```diff
@@ -5,10 +5,12 @@
 
 .ui.flashed.message,
 .ui.form .flashed.message {
-  z-index: 0;
   &.manage {
     padding: 1em 0;
   }
+  &.record-banner {
```

Comment:
why is a new class needed? can't we apply this to the flashed message directly?

Reply:
also, shouldn't it be in invenio-theme?

Reply:
ok sorry, it took me some time to understand the implications.
if I managed to understand correctly, the fix will work only for the record detail page, what if we have the banner in other pages? f.e. community form has an error?
if that is the case could you propose a solution which is more generic and works for all cases, not only for a record?